docs/docs/FAQ.mdx

```diff
@@ -162,6 +163,17 @@ We haven't implemented an official mechanism for creating albums from external l
 
 Duplicate checking only exists for upload libraries, using the file hash. Furthermore, duplicate checking is not global, but _per library_. Therefore, a situation where the same file appears twice in the timeline is possible, especially for external libraries.
 
+### Why are my edits to files not being saved in read-only external libraries?
+
+In external libraries mounted as read-only (`:ro`), Immich is unable to create the `.xmp` sidecar files to store edited file metadata.
+For this reason, the metadata (timestamp, location, description, star rating, etc.) cannot be edited for files in read-only external libraries.
+Images in read-write external libraries (the default) can be edited as normal.
+
+### How are deletions of files handled in external libraries?
+
+Immich will attempt to delete original files when the trash is emptied. In read-write external libraries (the default), Immich will delete the original file.
+In read-only libraries, files can be trashed in the web UI, but when the trash is emptied, they will re-appear in the main timeline since Immich is unable to delete the original file.
```

Comment:
Maybe clarify what a read-only library is in parantheses here too to remove any ambiguity.